modules/Microsoft.Storage/storageAccounts/blobServices/deploy.bicep

```diff
@@ -2,14 +2,52 @@
 @description('Conditional. The name of the parent Storage Account. Required if the template is used in a standalone deployment.')
 param storageAccountName string
 
-@description('Optional. Indicates whether DeleteRetentionPolicy is enabled for the Blob service.')
-param deleteRetentionPolicy bool = true
+@description('Optional. Automatic Snapshot is enabled if set to true.')
+param automaticSnapshotPolicyEnabled bool = false
+
+@description('Optional. The blob service properties for change feed events. Indicates whether change feed event logging is enabled for the Blob service.')
+param changeFeedEnabled bool = true
+
+@minValue(0)
+@maxValue(146000)
+@description('Optional. Indicates whether change feed event logging is enabled for the Blob service. Indicates the duration of changeFeed retention in days. Minimum value is 1 day and maximum value is 146000 days (400 years). A "0" value indicates an infinite retention of the change feed.')
+param changeFeedRetentionInDays int = 7
+
+@description('Optional. The blob service properties for container soft delete. Indicates whether DeleteRetentionPolicy is enabled.')
+param containerDeleteRetentionPolicyEnabled bool = true
+
+@minValue(1)
+@maxValue(365)
+@description('Optional. Indicates the number of days that the deleted item should be retained. The minimum specified value can be 1 and the maximum value can be 365.')
+param containerDeleteRetentionPolicyDays int = 7
+
+@description('Optional. This property when set to true allows deletion of the soft deleted blob versions and snapshots. This property cannot be used blob restore policy. This property only applies to blob service and does not apply to containers or file share.')
+param containerDeleteRetentionPolicyAllowPermanentDelete bool = false
+
+@description('Optional. Specifies CORS rules for the Blob service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Blob service.')
+param corsRules array = []
+
+@description('Optional. Indicates the default version to use for requests to the Blob service if an incoming request\'s version is not specified. Possible values include version 2008-10-27 and all more recent versions.')
+param defaultServiceVersion string = ''
+
+@description('Optional. The blob service properties for blob soft delete. Indicates whether DeleteRetentionPolicy is enabled for the Blob service.')
+param deleteRetentionPolicyEnabled bool = true
 
 @description('Optional. Indicates the number of days that the deleted blob should be retained. The minimum specified value can be 1 and the maximum value can be 365.')
 param deleteRetentionPolicyDays int = 7
```

Comment:
We should remove the limits from the description and use the Metadata limitations instead

Reply:
Thanks. Added metadata, removed information from description and generated readme after applying suggestions